code/modules/mob/living/simple_animal/abnormality/teth/ppodae.dm

```diff
@@ -0,0 +1,176 @@
+/mob/living/simple_animal/hostile/abnormality/ppodae
+	name = "Ppodae"
+	desc = "The Goodest Boy in the World"
+	icon = 'icons/mob/ppodae.dmi'
+	icon_state = "ppodae"
+	icon_living = "ppodae"
+	maxHealth = 400 //fast but low hp abno
+	health = 400
+	threat_level = TETH_LEVEL
+	move_to_delay = 1
+	faction = list("hostile")
+	response_help_continuous = "pet"
+	response_help_simple = "pet"
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = 60,
+						ABNORMALITY_WORK_INSIGHT = list(40, 40, 30, 30, 30),
+						ABNORMALITY_WORK_ATTACHMENT = 40,
+						ABNORMALITY_WORK_REPRESSION = list(40, 40, 30, 30, 30),
+						)
+	work_damage_amount = 4
+	work_damage_type = RED_DAMAGE
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1.5, WHITE_DAMAGE = 0.8, BLACK_DAMAGE = 1, PALE_DAMAGE = 2)
+	can_breach = TRUE
+	start_qliphoth = 2
+	vision_range = 14
+	aggro_vision_range = 20
+	stat_attack = HARD_CRIT
+	var/smash_length = 2
+	var/smash_width = 1
+	var/can_act = TRUE
+
+	var/turf/ppodae_location
```

Comment:
This is totally not needed. You can just get the landmark's location from datum_reference.

Reply:
Done